vscode/src/fetch.node.ts

```diff
@@ -8,6 +7,9 @@ import type { Configuration } from '@sourcegraph/cody-shared'
 import { agent } from '@sourcegraph/cody-shared/src/fetch'
 import { getConfiguration } from './configuration'
 
+import { HttpProxyAgent } from 'http-proxy-agent';
+import {HttpsProxyAgent} from "https-proxy-agent";
```

Comment:
Apparently the formatter didn't fix this 🤷 

```suggestion
import { HttpsProxyAgent } from "https-proxy-agent";
```